### PR TITLE
fixing compact responders output

### PR DESCRIPTION
**Pull request by jmervine**
1. Fixing compact responders output, addresses #153
2. Fixing occasional spec error (thanks @lexelby)
3. Added a test to ensure preceeding "," isn't present in compact responders output.

@lexelby @joshuatobin

### CODE REVIEW

**Comment by lexelby** [comment]
:+1:

File: spec/factories/user.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:name)   { |n| "User#{n} Lastname" }
-    sequence(:handle) { |n| "user#{n}" }
-    sequence(:email)  { |n| "user#{n}@example.com" }
+    sequence(:name)   { |n| "Name#{n} Lastname" }
+    sequence(:handle) { |n| "handle#{n}" }
+    sequence(:email)  { |n| "email#{n}@example.com" }
```
Comment (joshuatobin) [inline spec/factories/user.rb]: Might be nice to leave a comment here.

**Comment by joshuatobin** [comment]
👍

**Comment by jmervine** [comment]
@lexelby I got another racy failure [here](https://travis-ci.org/heroku/retrodot/builds/185833799). I'm pretty sure [these changes to the process helper](https://github.com/heroku/retrodot/pull/158/commits/c11596b9ae2664ce355a245fa563cf9e883bcbdb) will fix it for all time though.

cc @joshuatobin

**Pull request merged**